### DISCUSSION TITLE
schema

### DISCUSSION THREAD

**Discussion by rajamoulimallareddy** [prisma/prisma#17727] [Q&A]
model Guild {
  guildId       String        @id
  prefix        String        @default("t!")
  autoplay      Boolean       @default(false)
  djroles       String[]
  announcements Boolean       @default(false)
  setvc         String[]
  ban           String[]
  voiceId       String?
  textId        String?
  requester     Boolean       @default(false)
  buttons       Boolean       @default(false)
  twenty4seven  Boolean       @default(false)
  language      String        @default("en")
  premium       PremiumModel?
  guildSetup    guildSetup?   @relation(fields: [guildSetupId], references: [id])
  guildSetupId  Int?
}

model PremiumModel {
  id               Int     @id @default(autoincrement())
  active           Boolean @default(false)
  expiry           String?
  subtype_monthly  Boolean @default(false)
  subtype_lifetime Boolean @default(false)
  Guild            Guild?  @relation(fields: [guildGuildId], references: [guildId])
  guildGuildId     String? @unique
}

model guildSetup {
  id            Int     @id @default(autoincrement())
  channelId     String?
  lastMessageId String?
  Guild         Guild[]
}

i used `this.client.prisma.guild.create({ data: { guildId: msg.guild.id, guildSetup: { create: { channelId: msg.channel.id, lastMessageId:msg.channel.id } } } })`

and getting this how do i fix this 
![image](https://user-images.githubusercontent.com/68652418/216590515-ae81df74-f255-4bc1-be08-ac0455586edb.png)

**Comment by nurul3101** [comment]
Hey @rajamoulimallareddy 👋 
Can you please check what is the value of `guildId` that you are passing in the create function?
Does a record with the provided `guildId` already exists?

**Reply by rajamoulimallareddy** [reply]
ya my bad. and also how do i find the guildSetup using the Guild Model?

**Reply by nurul3101** [reply]
Do you mean you want to include guildSetup when fetching a guild record?

**Reply by rajamoulimallareddy** [reply]
yes

**Reply by rajamoulimallareddy** [reply]
hey there how to do that?

**Reply by nurul3101** [reply]
This query should include guildSetup details while fetching a guild
```ts
await prisma.guild.findUnique({
    where: {
      guildId: '123',
    },
    include: {
      guildSetup: true,
    },
  });
```
For reference: [Include docs](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#include)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
